docker-entrypoint.sh

```diff
@@ -7,6 +7,9 @@ window.OIDC_CLIENT_SECRET="${OIDC_CLIENT_SECRET:-undefined}";
 window.OIDC_REDIRECT_URI="${OIDC_REDIRECT_URI:-undefined}";
 EOF
 
-sed -i".old" 's/<body>/<body><script type="text\/javascript" src="\/config.js"><\/script>/' /usr/share/nginx/html/index.html
+sed -i".old" 's#<body>#<body><script type="text/javascript" src="/config.js"></script>#' /usr/share/nginx/html/index.html
+
+sed -i".old" 's#<head>#<head><title>'${APP_TITLE:-LiqoDash}'</title>#' /usr/share/nginx/html/index.html
+sed -i".old" 's#<head>#<head><link rel="icon" href="'${APP_FAVICON:-/logo_4.png}'">#' /usr/share/nginx/html/index.html
```

Comment:
Just my two cents... What about performing each sed operation only if the corresponding variable is defined? To prevent hardcoding `LiqoDash` and `logo_4`, which may lead to hard to spot bugs if those are a modified in webpack